packages/flutter/lib/src/foundation/annotations.dart

```diff
@@ -0,0 +1,107 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// A category with which to annotate a class, for documentation
+/// purposes.
+///
+/// A category is usually represented as a section and a subsection, each
+/// of which is a string. The project to which the class belongs defines the
```

Comment:
library is a defined term. Package is a defined term. Either of those might work?

Or, maybe more simply, "The author of the class defines the categories"... put the responsibility on the author.

(I didn't know if you meant IntelliJ project, or "logical" project, or a user's project?)